applications/ConstitutiveLawsApplication/custom_constitutive_thermal/small_strains/elastic/thermal_linear_plane_strain.h

```diff
@@ -0,0 +1,268 @@
+// KRATOS ___                _   _ _         _   _             __                       _
+//       / __\___  _ __  ___| |_(_) |_ _   _| |_(_)_   _____  / /  __ ___      _____   /_\  _ __  _ __
+//      / /  / _ \| '_ \/ __| __| | __| | | | __| \ \ / / _ \/ /  / _` \ \ /\ / / __| //_\\| '_ \| '_  |
+//     / /__| (_) | | | \__ \ |_| | |_| |_| | |_| |\ V /  __/ /__| (_| |\ V  V /\__ \/  _  \ |_) | |_) |
+//     \____/\___/|_| |_|___/\__|_|\__|\__,_|\__|_| \_/ \___\____/\__,_| \_/\_/ |___/\_/ \_/ .__/| .__/
+//                                                                                         |_|   |_|
+//
+//  License:         BSD License
+//                   license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//
+
+# pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_constitutive/linear_plane_strain.h"
+#include "custom_utilities/advanced_constitutive_law_utilities.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class ThermalLinearPlaneStrain
+ * @ingroup StructuralMechanicsApplication
+ * @brief This class defines a Thermo dependant CL, including the addition of thermal expansion strains
+ * @details This class derives from the linear elastic case on 3D
+ * @author Alejandro Cornejo
+ */
+class KRATOS_API(CONSTITUTIVE_LAWS_APPLICATION) ThermalLinearPlaneStrain
+    : public LinearPlaneStrain
+{
+public:
+
+    ///@name Type Definitions
+    ///@{
+
+    typedef LinearPlaneStrain BaseType;
+
+    /// Counted pointer of LinearPlaneStrain
+    KRATOS_CLASS_POINTER_DEFINITION(ThermalLinearPlaneStrain);
+
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief Default constructor.
+     */
+    ThermalLinearPlaneStrain() 
+    {
+
+    }
```

Comment:
You can let the compiler to add it for you.

Reply:
Apparently in windows it fails to compile if I remove them...

Reply:
And if you do it like this
````c++
ThermalLinearPlaneStrain() = default;
````